pgrx/src/callbacks.rs

```diff
@@ -57,18 +57,17 @@ pub enum PgXactCallbackEvent {
 }
 
 impl PgXactCallbackEvent {
-    fn translate_pg_event(pg_event: pg_sys::XactEvent) -> Self {
+    fn translate_pg_event(pg_event: pg_sys::XactEvent::Type) -> Self {
+        use pg_sys::XactEvent::*;
         match pg_event {
-            pg_sys::XactEvent_XACT_EVENT_ABORT => PgXactCallbackEvent::Abort,
-            pg_sys::XactEvent_XACT_EVENT_COMMIT => PgXactCallbackEvent::Commit,
-            pg_sys::XactEvent_XACT_EVENT_PARALLEL_ABORT => PgXactCallbackEvent::ParallelAbort,
-            pg_sys::XactEvent_XACT_EVENT_PARALLEL_COMMIT => PgXactCallbackEvent::ParallelCommit,
-            pg_sys::XactEvent_XACT_EVENT_PARALLEL_PRE_COMMIT => {
-                PgXactCallbackEvent::ParallelPreCommit
-            }
-            pg_sys::XactEvent_XACT_EVENT_PREPARE => PgXactCallbackEvent::Prepare,
-            pg_sys::XactEvent_XACT_EVENT_PRE_COMMIT => PgXactCallbackEvent::PreCommit,
-            pg_sys::XactEvent_XACT_EVENT_PRE_PREPARE => PgXactCallbackEvent::PrePrepare,
+            XACT_EVENT_ABORT => PgXactCallbackEvent::Abort,
+            XACT_EVENT_COMMIT => PgXactCallbackEvent::Commit,
+            XACT_EVENT_PARALLEL_ABORT => PgXactCallbackEvent::ParallelAbort,
+            XACT_EVENT_PARALLEL_COMMIT => PgXactCallbackEvent::ParallelCommit,
+            XACT_EVENT_PARALLEL_PRE_COMMIT => PgXactCallbackEvent::ParallelPreCommit,
+            XACT_EVENT_PREPARE => PgXactCallbackEvent::Prepare,
+            XACT_EVENT_PRE_COMMIT => PgXactCallbackEvent::PreCommit,
+            XACT_EVENT_PRE_PREPARE => PgXactCallbackEvent::PrePrepare,
```

Comment:
Making these matches a bit less awful to read was basically my primary goal here.